README.md

```diff
@@ -3,65 +3,84 @@
+    - Windows: `powershell -command "Get-ComputerInfo`
+  
+If you do not include this information, we cannot help you.
 
+
+## Contents
 - [Prerequisites](#prerequisites)
-- [Setup](#setup)
+- [Environment Setup](#environment-setup)
 - [Quickstart](#quickstart)
-- [Compute Node Architecture](#compute-node-architecture)
-- [Docker Example Explained](#docker-example)
+- [Testing](#testing)
 - [Commands](#commands)
-- [Troubleshooting](#troubleshooting)
-- [Prerequisites Help](#prerequisites-help)
 - [Uninstall](#uninstall)
 
-## Prerequisites
 
-All you need is:
+## Prerequisites
+1.  An [AWS (Amazon Web Services) Account](https://portal.aws.amazon.com/billing/signup)
+2.  AWS [Billing set up](https://console.aws.amazon.com/billing/home?#/paymentmethods)
+3.  AWS [Access Keys](https://console.aws.amazon.com/iam/home?#/security_credentials)
+4.  [Numerai API Keys](https://numer.ai/account)
+5.  Python and Docker (see [Environment Setup](#environment-setup))
 
-1.  AWS (Amazon Web Services) account setup 
-2.  AWS API keys (Access key ID and Secret Access Key)
-3.  Numerai API Key (Public ID and secret key)
+Our friend Arbitrage created [this tutorial](https://www.youtube.com/watch?v=YFgXMpQszpM&feature=youtu.be)
+to help you get set up with your AWS/Numerai API Keys
 
 
-## Setup
+## Environment Setup
 
 For your convenience, we've included setup scripts in the `scripts` directory that will ensure the prerequisites are installed.
 You can download and run the setup script for your OS with one of the following commands:
 
-- Mac Terminal(cmd + space, type `terminal`, select `terminal.app`):
+- Mac Terminal (cmd + space, type `terminal`, select `terminal.app`):
     ```
-    curl https://raw.githubusercontent.com/numerai/numerai-cli/ndharasz/setup-scripts/scripts/setup-mac.sh | bash
+    curl https://raw.githubusercontent.com/numerai/numerai-cli/master/scripts/setup-mac.sh | bash
     ```
 
 - Ubuntu 18 Terminal (ctrl + alt + t):
     ```
-    apt update && apt install -y libcurl4 curl && curl https://raw.githubusercontent.com/numerai/numerai-cli/ndharasz/setup-scripts/scripts/setup-ubu18.sh | bash
+    apt update && apt install -y libcurl4 curl && curl https://raw.githubusercontent.com/numerai/numerai-cli/master/scripts/setup-ubu18.sh | bash
     ```
 
 - Windows Command Prompt (windows key, type `cmd`, select Command Prompt):
     ```
-    powershell -command "$Script = Invoke-WebRequest 'https://raw.githubusercontent.com/numerai/numerai-cli/ndharasz/setup-scripts/scripts/setup-win10.ps1'; $ScriptBlock = [ScriptBlock]::Create($Script.Content); Invoke-Command -ScriptBlock $ScriptBlock"
+    powershell -command "$Script = Invoke-WebRequest 'https://raw.githubusercontent.com/numerai/numerai-cli/master/scripts/setup-win10.ps1'; $ScriptBlock = [ScriptBlock]::Create($Script.Content); Invoke-Command -ScriptBlock $ScriptBlock"
     ```
+If you run into issues running one of these scripts, please report immediately to Rocketchat followig.
```

Comment:
If you run into issues running one of these scripts, please report your issue in the Rocketchat #support channel

Reply:
We have a #compute channel that is linked above, that was the channel I was going to point them to